scripts/gha/retry_test_failures.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2023 Google LLC
```

Comment:
2024

Reply:
Eh, this is mostly a copy of the script from C++ that was written in 2023, should it still be 2024?